website/tests.py

```diff
@@ -131,3 +135,16 @@ def test_on_hide(self):
 
             if "hidden" not in filename:
                 self.assertFalse(True, "files rename failed")
+
+class IssueBaseCreateTest(APITestCase):
+    def test_throttle_exceed_limit(self):
+        for i in range(10000):
+            data = {'url': 'http://example.com', 'description': 'test', 'markdownInput': 'test', 'screenshots': 'test', 'captcha_1': 'PASSED'}
+            response = self.client.post('/report/', data)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Is this supposed to be a 200?

Reply:
Let me check again

Reply:
Since the test if for whether the form is valid or not

Reply:
The test should be if the rate limit was hit